src/OSPSuite.Core/Domain/Data/DataInfo.cs

```diff
@@ -61,12 +61,6 @@ public class DataInfo
       /// </summary>
       public DateTime Date { get; set; }
 
-      /// <summary>
-      ///    For observed data it is normally the name of the excel sheet or csv file. For simulated data,
-      ///    it could be the name of the simulation
-      /// </summary>
-      public string Source { get; set; }
```

Comment:
Remove this from the Data Info.. Not only is it saved in the data repo, but also in each column!